JeffShepherd/qwixx-clone
# Feature/game

## What does this PR do?

- Establishes the basis of the Game component
- Removes unnecessary boilerplate code and other assets
- Add normalize css file to remove default css properties

## How should it be tested in the UI?

- Start application with 'npm run' and ensure there are not any console errors
- The full background of the screen should be red currently to represent the Game component
